blocked-edges/4.18.0-rc.0-MachineConfigServerCARotation.yaml

```
@@ -0,0 +1,8 @@
+to: 4.18.0-rc.0
+from: 4[.]18[.]0-ec[.]4[+].*
+url: https://issues.redhat.com/browse/MCO-1481
+name: MachineConfigServerCARotation
+message: |-
+  4.18.0-ec.4 adjusted machine-config server CA management.  The change was reverted in 4.18.0-rc.0, but clusters running 4.18.0-ec.4 need manual steps to be able to scale new Machines. 
```

nit: Clarifying what needs to be done and some minor changes. Feel free to ignore.
```suggestion
  The 4.18.0-ec.4 version adjusted the management of the machine-config server CA. The change was reverted in 4.18.0-rc.0, but clusters running 4.18.0-ec.4 need to perform manual steps to scale new Machines after updating to a new version.
```